synth.py

```diff
@@ -26,23 +26,43 @@
 
 # skip index, protos, package.json, and README.md
 s.copy(v1_admin_library, "dev", excludes=["package.json", "README.md", "src/index.ts", "src/v1/index.ts", 
-    "tsconfig.json", "tslint.json", "linkinator.config.json", "webpack.config.js"])
+    "tsconfig.json", "linkinator.config.json", "webpack.config.js"])
 s.copy(v1beta1_library, "dev", excludes=["package.json", "README.md", "src/index.ts", "src/v1beta1/index.ts",
-    "tsconfig.json", "tslint.json", "linkinator.config.json", "webpack.config.js"])
+    "tsconfig.json", "linkinator.config.json", "webpack.config.js"])
 s.copy(v1_library, "dev", excludes=["package.json", "README.md", "src/index.ts", "src/v1/index.ts",
-    "tsconfig.json", "tslint.json", "linkinator.config.json", "webpack.config.js"])
+    "tsconfig.json", "linkinator.config.json", "webpack.config.js"])
 
 # Fix dropping of google-cloud-resource-header
 # See: https://github.com/googleapis/nodejs-firestore/pull/375
 s.replace(
     "dev/src/v1beta1/firestore_client.ts",
-    "return this\._innerApiCalls\.listen\(options\);",
-    "return this._innerApiCalls.listen({}, options);",
+    "return this\.innerApiCalls\.listen\(options\);",
```

Comment:
Every time I see this replacement I wonder what's wrong with the code here, why we need this replacement?

Reply:
I do think the generated code is busted here. We use `options` to set an additional header, but the way this gets passed down here is as the request (if I remember this correctly). If we don't modify the code, the header is dropped. This only affects bidirectional streaming APIs with additional metadata, so probably just us. I do think we should fix this upstream at some point.

Reply:
OK, this sounds like a bug - filed https://github.com/googleapis/gapic-generator-typescript/issues/425 and one of us will investigate.